config/esp32/components/chip/CMakeLists.txt

```diff
@@ -173,10 +173,6 @@ if (CONFIG_BUILD_CHIP_TESTS)
     chip_gn_arg_bool("chip_build_tests"     "true")
 endif()
 
-if (CONFIG_IM_PRETTY_PRINT)
```

Comment:
Is it present somewhere in the Kconfig files for the platform?

Reply:
No its not.

`CONFIG_CHIP_CONFIG_IM_PRETTY_PRINT` is in Kconfig and that is being used in CMakeLists.txt

https://github.com/project-chip/connectedhomeip/blob/ff5490ab54b9e68746bdc5ea853a08a0425be1f2/config/esp32/components/chip/Kconfig#L1006-L1012

https://github.com/project-chip/connectedhomeip/blob/ff5490ab54b9e68746bdc5ea853a08a0425be1f2/config/esp32/components/chip/CMakeLists.txt#L81-L83

Check for `CONFIG_IM_PRETTY_PRINT` in CMakeLists.txt is just a dead code 

https://github.com/project-chip/connectedhomeip/blob/ff5490ab54b9e68746bdc5ea853a08a0425be1f2/config/esp32/components/chip/CMakeLists.txt#L176-L178